src/components/item-info/index.js

```diff
@@ -22,13 +23,18 @@ export const ItemInfo = ({
   feed,
   supply,
   isDetailView,
-  restricted,
-  ban,
 }) => {
   const { syncTaquito, collect, fulfillObjktcomAsk, curate, acc } =
     useContext(HicetnuncContext)
 
+  const { walletBlockList } = useSettings()
+
   const [showSignStatus, setShowSignStatus] = useState(false)
+  const [restricted, setRestricted] = useState(false)
+
+  useEffect(() => {
```

Comment:
imo `useMemo` is the more fitting hook in this case. `restricted` is basically a derivation of `walletBlockList` and `creator`.